serverside/zerotrace/example/main.go

```diff
@@ -226,7 +284,7 @@ func serveFormTemplate(w http.ResponseWriter) {
 
 func main() {
 	var logfilePath, addr, domain string
-	flag.StringVar(&iface, "iface", "enp1s0f1", "Network interface name to listen on (default: eth0)")
```

Comment:
I miss the times where interfaces had easy-to-remember names!

Reply:
Same!